Dockerfile

```diff
@@ -1,17 +1,16 @@
 FROM socrata/base
 
-RUN apt-get update && apt-get -y install libfontconfig wget adduser openssl ca-certificates
+ENV GRAFANA_VERSION 2.5.0
 
-RUN wget http://grafanarel.s3.amazonaws.com/builds/grafana_latest_amd64.deb
-
-RUN dpkg -i grafana_latest_amd64.deb
+RUN apt-get update && \
+    apt-get -y install libfontconfig wget adduser openssl ca-certificates && \
+    apt-get clean && \
+    wget https://grafanarel.s3.amazonaws.com/builds/grafana_${GRAFANA_VERSION}_amd64.deb -O /tmp/grafana.deb && \
+    dpkg -i /tmp/grafana.deb && \
+    rm /tmp/grafana.deb
 
 ADD ship.d /etc/ship.d
 
-EXPOSE 3000
-
-VOLUME ["/var/lib/grafana"]
-VOLUME ["/var/log/grafana"]
-VOLUME ["/etc/grafana"]
+VOLUME ["/var/lib/grafana", "/var/log/grafana", "/etc/grafana"]
```

Comment:
I know this isn't part of your change but ... what are these `VOLUME` additions actually doing?  It doesn't seem like exposing those locations on the host to the container is at all useful in our world unless I'm missing something.